raster-foundry/blasterjs
# Upgrade nodejs to latest LTS, upgrade Polished

## Overview

- upgrades node to the latest LTS and removes webpack-blocks, which is no longer needed now that we've moved to docz
- upgrades polished to the latest version

### Checklist


## Testing Instructions

 * `nvm use` and verify the node version is v10.15.3
 * `npm install` and `npm run start` and see that everything still works
 * Check `button`, `progressBar`, and `table` since they used polished

Closes #174 
Closes #68
